backend/audit/intakelib/common/util.py

```diff
@@ -73,7 +73,7 @@ def is_yes_or_no_or_gsa_migration(v):
     # in historic data migration to pass validation.
     value = str(v).strip()
 
-    return value in ["Y", "N", settings.GSA_MIGRATION]
+    return value in ["Y", "y", "N", "n", settings.GSA_MIGRATION]
```

Comment:
We don't want to allow y/n here, as these are not expected at the schema level. In general, our Python checks are/should be less permissive than JSONSchema checks to prevent data from passing the former and failing the latter.

Reply:
Good call, done